src/MLJFlux.jl

```diff
@@ -31,8 +33,7 @@ include("mlj_model_interface.jl")
 export NeuralNetworkRegressor, MultitargetNeuralNetworkRegressor
 export NeuralNetworkClassifier, NeuralNetworkBinaryClassifier, ImageClassifier
 export CUDALibs, CPU1
+export EntityEmbedder
```

Comment:
Only export if the average  user is going to need this. Why would they need to reference `EntityEmbedder`?

Reply:
Done.